crates/jstz_node/src/services/logs/db.rs

```diff
@@ -130,17 +132,30 @@ impl Db {
         mut stmt: Statement<'_>,
         params: P,
     ) -> QueryResponseResult {
-        let logs = stmt
+        let query_result = stmt
             .query_map(params, |row| {
-                Ok(QueryResponse::Log {
-                    level: row.get(1)?,
-                    content: row.get(2)?,
-                    function_address: row.get(3)?,
-                    request_id: row.get(4)?,
-                })
+                Ok((
+                    row.get::<usize, String>(0)?,
```

Comment:
Why are the indices changed to `0, 1, 2, 3`? Did the log table's schema change?

Reply:
My bad, fixed